mllib/src/test/scala/org/apache/spark/ml/evaluation/RegressionEvaluatorSuite.scala

```diff
@@ -65,15 +65,15 @@ class RegressionEvaluatorSuite
 
     // default = rmse
     val evaluator = new RegressionEvaluator()
-    assert(evaluator.evaluate(predictions) ~== 0.1019382 absTol 0.001)
+    assert(evaluator.evaluate(predictions) ~== 0.1013829 absTol 0.001)
```

Comment:
It can't quite be based on the R results if the input is no longer the same; even if it were it's not clear the thresholds were set in any principled way, just looking at them. 

I can see the argument that the tolerance should stay the same, because we have a different but equally fixed new input. So, meh, toss up.